src/gui/tray/TrayFolderListItem.qml

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) by Oleksandr Zolotov <alex@nextcloud.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * for more details.
+ */
+import QtQml 2.15
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import Style 1.0
+
+MenuItem {
+    id: root
+
+    property string title: ""
```

Comment:
cannot you simply use the existing `text` property ?
would be less custom
we already have existing custom components that are uneasy to use and may need resources to be maintained

Reply:
> cannot you simply use the existing `text` property ? would be less custom we already have existing custom components that are uneasy to use and may need resources to be maintained

@mgallien would not we then have extra text displayed using the built-in `MenuItem` label?